apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -88,7 +94,23 @@ export default {
   },
   computed: {
     title() {
-      return this.titlePrefix + " " + this.tableName;
+      return `${this.titlePrefix} ${this.tableName} ${this.pageName}`;
+    },
+    pageName() {
+      if (this.currentPage > 1) {
+        const pageHeadings = this.tableMetaData.columns.reduce(
+          (accum, column) => {
+            if (column.columnType === "HEADING") {
+              accum.push(column.name);
+            }
+            return accum;
+          },
+          []
```

Comment:
Would be nice to put this into a global or in data, so it's calculated only once.